### PR TITLE
fixed: allowed dateformat YYYY-MM-DD and YYYY/MM/DD

### PR DESCRIPTION
**Pull request by tsunoda-takahiro**
Enhanced date validation to support both YYYY-MM-DD and YYYY/MM/DD

### CODE REVIEW

File: api_v1/serializers.py [hunk 1]
```diff
@@ -187,11 +187,15 @@ def validate_named_attr(value):
...
-                try:
-                    datetime.strptime(value, "%Y-%m-%d")
-                except ValueError:
-                    raise ValueError("Incorrect data format, should be YYYY-MM-DD")
-                return datetime.strptime(value, "%Y-%m-%d")
+                date_formats = ["%Y-%m-%d", "%Y/%m/%d"]  # List of acceptable date formats
+                for date_format in date_formats:
+                    try:
+                        parsed_date = datetime.strptime(value, date_format)
+                        return parsed_date
```
Comment (userlocalhost) [inline api_v1/serializers.py]: This implementation is good and this explanatory variable is useful to understand mechanism in a method.
(c.f. https://qiita.com/KNR109/items/3b14e2e8f89a33c0f959#%E8%AA%AC%E6%98%8E%E5%A4%89%E6%95%B0%E3%82%92%E5%88%A9%E7%94%A8%E3%81%99%E3%82%8B)

But, some time these explanatory variables seems to be redundant. In that case, it's better to write it simply like
```
return datetime.strptime(value, date_format)
```

File: api_v1/tests/test_api.py [hunk 2]
```diff
@@ -1398,3 +1398,43 @@ def test_update_entry_when_trigger_is_set(self):
...
+
+        # Test for date formatted as YYYY-MM-DD
+        params["attrs"]["date"] = "2018-12-31"
+        resp = self.client.post("/api/v1/entry", json.dumps(params), "application/json")
+        self.assertEqual(resp.status_code, 200)
+
+        # Test for invalid date formatted as YYYY/1/31
+        params["attrs"]["date"] = "201/12/31"
+        resp = self.client.post("/api/v1/entry", json.dumps(params), "application/json")
+        self.assertEqual(resp.status_code, 400)
```
Comment (userlocalhost) [inline api_v1/tests/test_api.py:1440]: It's good to cover whole cases 👍

**Review by userlocalhost — APPROVED** [review]
Thank you very much for enhancing its feature 👍 
I hope to blush its implementation up. But it's OK to merge this without changing it.

**Comment by tsunoda-takahiro** [comment]
I have made the suggested corrections.

**Review by hinashi — APPROVED** [review]
LGTM!

**Pull request merged**